packages/core/src/components/popover/popover.tsx

```diff
@@ -52,12 +52,6 @@ export interface IPopoverProps extends IOverlayableProps, IProps {
     arrowSize?: number;
 
     /**
-     * Constraints for the underlying Tether instance.
-     * See http://tether.io/#constraints.
-     */
-    constraints?: TetherUtils.ITetherConstraint[];
```

Comment:
Oops, didn't stop to think about that. I've reimplemented this again with `@deprecated`